VotingApplication/VotingApplication.Web/Scripts/Controllers/ManagePageController.js

```diff
@@ -0,0 +1,33 @@
+﻿(function () {
+    angular.module('GVA.Creation').controller('ManagePageController', ['$scope', '$routeParams', 'AccountService', 'ManageService',
```

Comment:
This is rather a long line. Would be a bit easier on the eyes if .module and .controller were each on their own line.